setup.cfg

```diff
@@ -58,6 +58,7 @@ testing =
     pytest-cov>=2.10
     papermill
     jupyter
+    seaborn
```

Comment:
I'm confused what is breaking without seaborn installed. As far as I know, we don't use it at all in the lib, and I can't find the file you mentioned (`test_dataset.py`).

Reply:
my bad, I thought `test_dataset.py` as part of the ESC50 package belongs to the `fastaudio` library. 
tests are still passing in CI with the change reverted so seems to be related to my local setup